cmd/swarm/feeds_test.go

```diff
@@ -162,4 +161,36 @@ func TestCLIFeedUpdate(t *testing.T) {
 	if !bytes.Equal(data, retrieved) {
 		t.Fatalf("Received %s, expected %s", retrieved, data)
 	}
+
+	// test publishing a manifest for a different user
+	flags = []string{
+		"--bzzapi", srv.URL,
+		"feed", "create",
+		"--topic", topic.Hex(),
+		"--user", "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", // different user
+	}
+
+	log.Info("Publishing manifest with 'swarm feed create' for a different user")
+	cmd = runSwarm(t, flags...)
+	_, matches = cmd.ExpectRegexp(`[a-f\d]{64}`) // regex hack to extract stdout
+	cmd.ExpectExit()
```

Comment:
shouldn't we also expect a non-zero exit code here?

Reply:
Nope, this is a legitimate command.

I note there is quite a bit of confusion about manifest creation.
You can create a feed manifest without a private key because a feed manifest is just a JSON with the topic + address like below, uploaded to `bzz-raw:`. Thus, `swarm feed create --topic 0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb --user 0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa` produces this manifest:
```json
{
  "entries": [
    {
      "contentType": "application/bzz-feed",
      "mod_time": "0001-01-01T00:00:00Z",
      "feed": {
        "topic": "0xbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb",
        "user": "0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
      }
    }
  ]
}

```
... just a JSON uploaded like that. No need for a PK. The reason why we don't require a PK is because there is no need of a PK to build a manifest. Anyone can do it and you can't stop them. And it allows you to easily create manifests for yourself maybe in another computer where you don't have your PK at hand.

Reply:
See my comment above please, but I agree, there's no reason to attempt to disallow the manifest creation as long as there is no risk of detrimental effects of spoofing.

Reply:
@justelad I understand. This concept is so new I don't have experience in what is the best way to explain this. I am sure we will get better over time. As soon as I come up with something easier I'll propose a change, or we can open a discussion to see what is the best option. Again, this would be out of scope for this PR, whose humble purpose was to have an error message appear ;-)

Reply:
fair enough :)